### PR TITLE
Implement first version of issue and pull-request templates

### LINKED ISSUE #47: Create issue and pull-request templates

### PR DESCRIPTION
**Pull request by lulivi**
Closes #47

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+title: '[BUG]'
+labels: bug, unconfirmed
+assignees: ''
+---
+
+<!-- :warning: Please, try to follow the template -->
+
+### System information
+
+- **Script affected**:
```
Comment (fgaranda) [inline .github/ISSUE_TEMPLATE/bug.md:13]:
```suggestion
- **Affected script**:
```

File: .github/ISSUE_TEMPLATE/bug.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+- **Repository affeced**:
```
Comment (fgaranda) [inline .github/ISSUE_TEMPLATE/bug.md:14]:
```suggestion
- **Affected repository**:
```

File: .github/ISSUE_TEMPLATE/find-package.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+|      **Connext version(s)** | `--` |
+| **Connext architecture(s)** | `--` |
+
+### Description
+
+<!-- :warning: Provide a detailed description of the issue itself. -->
+
+### Steps to reproduce the issue
+
+<!-- :warning: List the steps necessary to face the issue. E.g.:
```
Comment (fgaranda) [inline .github/ISSUE_TEMPLATE/find-package.md:37]:
```suggestion
<!-- :warning: List the steps necessary to reproduce the issue. E.g.:
```

File: .github/ISSUE_TEMPLATE/find-package.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+1. Set X architecture environment variables.
+2. Create a build directory in the example X.
```
Comment (fgaranda) [inline .github/ISSUE_TEMPLATE/find-package.md:40]:
```suggestion
2. Create a build directory in example X.
```

File: .github/ISSUE_TEMPLATE/find-package.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+3. Run the following command:
+   ```
+   cmake .. -DCONNEXTDDS_DIR=/home/user/rti_connext_dds-6.1.1/
```
Comment (fgaranda) [inline .github/ISSUE_TEMPLATE/find-package.md:43]:
```suggestion
   cmake .. -DCONNEXTDDS_DIR=/home/user/rti_connext_dds-x.y.z/
```

File: .github/ISSUE_TEMPLATE/find-package.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+   ```
+4. The problem will popout:
```
Comment (fgaranda) [inline .github/ISSUE_TEMPLATE/find-package.md:45]:
```suggestion
4. The problem will pop up:
```

File: .github/ISSUE_TEMPLATE/find-package.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+   ```
+   Error: Can't find the library Y
```
Comment (fgaranda) [inline .github/ISSUE_TEMPLATE/find-package.md:47]:
```suggestion
   Error: Can't find library Y
```

File: .github/PULL_REQUEST_TEMPLATE/general.md [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+
+<!-- :warning: A summary of the changes in the pull-request. -->
+
+### Comments
+
+<!-- :warning: Anything to highlight? -->
+
+### Logs
+
+<!-- :warning: Add one `details` tag for each log file required.
```
Comment (fgaranda) [inline .github/PULL_REQUEST_TEMPLATE/general.md:23]:
```suggestion
<!-- :warning: Add one `details` tag for each required log file.
```

**Review by fgaranda — CHANGES_REQUESTED** [review]

**Review by fgaranda — APPROVED** [review]

File: Jenkinsfile [hunk 9]
```diff
@@ -132,7 +170,7 @@ pipeline {
                             name: STAGE_NAME,
                             title: 'Downloading',
                             summary: ':arrow_down: Downloading RTI Connext DDS libraries...',
```
Comment (rkorte) [inline Jenkinsfile:172]: Is this text seen by customers? We are slowly removing "DDS" from "RTI Connext DDS," but not necessarily in code/directory names, etc. If these are the only instances, or if you want to, remove "DDS" from these. But if that makes things inconsistent, and searching/replacing all is too big a project, then never mind.

**Review by rkorte — APPROVED** [review]

File: Jenkinsfile (hunk 9 above)
Comment (lulivi) [reply Jenkinsfile:172]: Alright! I'll remove them! Yes, these messages are seen by the costumers.

**Review by rkorte — APPROVED** [review]

**Pull request merged**